dotCMS/src/main/java/com/dotmarketing/business/PermissionBitFactoryImpl.java

```diff
@@ -365,22 +371,22 @@
 	 * 1. The host id
 	 * 2. Parent folder like path E.G. '/about/%' pass '%' if you want all from the host
 	 */
-    private final String selectChildrenHTMLPageSQL =
+    private static final String selectChildrenHTMLPageSQL =
             "select distinct li.id from identifier li where" +
                 " li.asset_type='htmlpage' and li.host_inode = ? and li.parent_path like ?" +
             " UNION ALL" +
                 " SELECT distinct li.id FROM identifier li" +
                     " INNER JOIN contentlet lc ON (lc.identifier = li.id and li.asset_type = 'contentlet')" +
-                    " INNER JOIN structure ls ON (lc.structure_inode = ls.inode and ls.structuretype = " + Structure.STRUCTURE_TYPE_HTMLPAGE + ")" +
+                    " INNER JOIN structure ls ON (lc.structure_inode = ls.inode and ls.structuretype = " + BaseContentType.HTMLPAGE.getType() + ")" +
                     " AND li.host_inode = ? and li.parent_path like ?";
-    private final String selectChildrenHTMLPageOnPermissionsSQL =
+    private static final String selectChildrenHTMLPageOnPermissionsSQL =
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)